src/main/java/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package service.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.DataConverter;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int ZERO_ELEMENT = 0;
+    private static final int ONE_ELEMENT = 1;
+    private static final int TWO_ELEMENT = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> fileData) {
+        return fileData.stream()
+                .skip(1)
+                .map(this::createFruitTransactionInstance)
+                .collect(Collectors.toList());
+    }
+
+    private FruitTransaction createFruitTransactionInstance(String lineFromFile) {
+        FruitTransaction fruitTransaction = new FruitTransaction();
```

Comment:
It's a good practice to encapsulate the creation of objects in a separate method. However, it would be better if you used a constructor in the `FruitTransaction` class to set the fields instead of using setters. This way you can make your `FruitTransaction` objects immutable, which is a good practice in Java.